#!/usr/bin/env python
PACKAGE = "twitcher_connection"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("twitter_api_url", str_t, 0, "URL for the twitter API", "https://api.twitter.com/1.1/")